0000-template.md

```diff
@@ -1,15 +1,22 @@
 # Feature name
 
 * Proposal: [SE-NNNN](NNNN-filename.md)
-* Author: [Swift Developer](https://github.com/swiftdev)
+* Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
+* Review Manager: TBD
 * Status: **Awaiting review**
-* Review manager: TBD
+
+*During the review process, add the following fields as needed:*
+
+* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution/), [Additional Commentary](https://lists.swift.org/pipermail/swift-evolution/)
```

Comment:
That's an interesting suggestion. We decided to use Decision Notes and keep the header information related to the outcome of a review, since this information is usually added by the review manager.